### PR TITLE
Rename lambda to identifier

### PR DESCRIPTION
**Pull request by nkryuchkov**
Depends on https://github.com/bloxapp/ssv/pull/682

### CODE REVIEW

File: exporter/README.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ The following information will be stored and served by exporter:
     "message": {
       "type": 3,
       "round": 1,
-      "lambda": "...",
+      "identifier": "...",
```
Comment (nivBlox) [inline exporter/README.md:61]: make sure this is really the right key

File: ibft/storage/ibft_store.go [hunk 2]
```diff
@@ -34,7 +34,7 @@ var (
 	metricsHighestDecided = promauto.NewGaugeVec(prometheus.GaugeOpts{
 		Name: "ssv:validator:ibft_highest_decided",
 		Help: "The highest decided sequence number",
-	}, []string{"lambda", "pubKey"})
+	}, []string{"identifier", "pubKey"})
```
Comment (nivBlox) [inline ibft/storage/ibft_store.go:37]: need to align Grafana as well

File: monitoring/README.md [hunk 3]
```diff
@@ -38,9 +38,9 @@ METRICS_API_PORT=15000
 * `ssv:network:ibft_decided_messages_outbound{topic}` Count IBFT decided messages outbound
 * `ssv:network:ibft_messages_outbound{topic}` Count IBFT messages outbound
 * `ssv:network:net_messages_inbound{topic}` Count incoming network messages
-* `ssv:validator:ibft_highest_decided{lambda}` The highest decided sequence number
-* `ssv:validator:ibft_round{lambda}` IBFTs round
-* `ssv:validator:ibft_stage{lambda}` IBFTs stage
+* `ssv:validator:ibft_highest_decided{identifier}` The highest decided sequence number
```
Comment (nivBlox) [inline monitoring/README.md:41]: Please align grafana as well

File: protocol/v1/qbft/controller/metrics.go [hunk 4]
```diff
@@ -1,17 +1,18 @@
...
-	"log"
 )
 
 var (
 	// metricsCurrentSequence for current instance
 	metricsCurrentSequence = promauto.NewGaugeVec(prometheus.GaugeOpts{
 		Name: "ssv:validator:ibft_current_sequence",
 		Help: "The highest decided sequence number",
-	}, []string{"lambda", "pubKey"})
+	}, []string{"identifier", "pubKey"})
```
Comment (nivBlox) [inline protocol/v1/qbft/controller/metrics.go:15]: align with grafana

File: protocol/v1/qbft/instance/metrics.go [hunk 5]
```diff
@@ -11,11 +11,11 @@ var (
 	metricsIBFTStage = promauto.NewGaugeVec(prometheus.GaugeOpts{
 		Name: "ssv:validator:ibft_stage",
 		Help: "IBFTs stage",
-	}, []string{"lambda", "pubKey"})
+	}, []string{"identifier", "pubKey"})
```
Comment (nivBlox) [inline protocol/v1/qbft/instance/metrics.go:14]: align grafana

File: protocol/v1/qbft/metrics.go [hunk 6]
```diff
@@ -1,20 +1,21 @@
...
+
+	"github.com/bloxapp/ssv/protocol/v1/message"
 )
 
 var (
 	metricsDecidedSigners = promauto.NewGaugeVec(prometheus.GaugeOpts{
 		Name: "ssv:ibft:last_decided_signers",
 		Help: "The highest decided sequence number",
-	}, []string{"lambda", "pubKey", "nodeId"})
+	}, []string{"identifier", "pubKey", "nodeId"})
```
Comment (nivBlox) [inline protocol/v1/qbft/metrics.go:18]: Align grafana

File: protocol/v1/qbft/msgqueue/metrics.go [hunk 7]
```diff
@@ -1,16 +1,17 @@
...
 	"github.com/prometheus/client_golang/prometheus/promauto"
-	"log"
 )
 
 var (
 	metricsMsgQRatio = promauto.NewGaugeVec(prometheus.GaugeOpts{
 		Name: "ssv:ibft:msgq:ratio",
 		Help: "The messages ratio between pop and add",
-	}, []string{"lambda", "index_name", "msg_type", "consensus_type"})
+	}, []string{"identifier", "index_name", "msg_type", "consensus_type"})
```
Comment (nivBlox) [inline protocol/v1/qbft/msgqueue/metrics.go:14]: align grafana

**Review by nivBlox — CHANGES_REQUESTED** [review]

File: ibft/storage/ibft_store.go (hunk 2 above)
Comment (nkryuchkov) [reply ibft/storage/ibft_store.go:37]: @nivBlox Could you please share the project link where I need to change it as well?

Comment (nkryuchkov) [reply ibft/storage/ibft_store.go:37]: @nivBlox thanks, done

File: exporter/README.md (hunk 1 above)
Comment (nkryuchkov) [reply exporter/README.md]: @nivBlox it is, but the whole example is outdated, we don't have `seq_number` or `signer_ids` anymore and they are not the scope of this PR. I think that needs to be fixed in a PR for making sure that docs are up-to-date.

**Review by nivBlox — APPROVED** [review]

**Pull request merged**